controllers/contact-controller.js

```diff
@@ -311,7 +308,31 @@ const ContactController = sequelize => {
     return contact;
   };
 
-  return { searchContacts, getContacts, createContact, queries: { queryOnName, queryOnPhone } };
+  const getContact = async id => {
+    const contact = await Contact.findByPk(id);
+
+    if (!contact) {
+      const errorMessage = `Contact with id ${id} not found`;
+      throw createError(404, errorMessage);
+    }
+
+    return contact;
+  };
+
+  const connectToCase = async (contactId, caseId) => {
+    const contact = await getContact(contactId);
+    const updatedContact = await contact.update({ caseId });
```

Comment:
What happens if any of these endpoints that write information has a db failure?  What gets returned to the client?

Reply:
express will return `500 Internal Error` in case of any unhandled error.